src/components/CheckboxGroup/__test__/index.test.tsx

```diff
@@ -0,0 +1,74 @@
+// CheckboxGroup.test.tsx
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import CheckboxGroup, { ICheckboxOption } from "../index"; // Adjust to your actual path
+import "@testing-library/jest-dom";
+import { describe, it, expect, vi } from "vitest";
+import { APP_CATEGORY } from "@/constants/discover";
+
+// 测试用的 options 数组
+const options: ICheckboxOption[] = [
+  {
+    value: APP_CATEGORY.NEW,
+    label: "💰 New",
+  },
+  {
+    value: APP_CATEGORY.GAME,
+    label: "🎮 Game",
+  },
+  {
+    value: APP_CATEGORY.EARN,
+    label: "💰 Earn",
+  },
+];
+
```

Comment:
you already import the constants you can always use that you do not need to declare another one

Reply:
Replaced